src/dxvk/dxvk_context.cpp

```diff
@@ -4063,6 +4063,9 @@ namespace dxvk {
         attachment.view->imageInfo().layout,
         attachment.view->imageInfo().stages,
         attachment.view->imageInfo().access);
+
+      m_cmd->trackResource<DxvkAccess::None>(attachment.view);
```

Comment:
We don't need to track the views here since the GPU won't use them, just the image.

Reply:
Resolved.